src/test/java/api/loans/scenarios/ChangeDueDateTests.java

```diff
@@ -98,6 +100,43 @@ public void canManuallyChangeTheDueDateOfLoan() {
       loanInStorage.containsKey("checkoutServicePoint"), is(false));
   }
 
+  @Test
+  public void cannotManuallyChangeTheDueDateOfClaimedReturnedLoan() {
+    final InventoryItemResource item = itemsFixture.basedUponNod();
+
+    IndividualResource loan = loansFixture.checkOutByBarcode(item);
+
+    final ClaimItemReturnedRequestBuilder claimedItemBuilder =
+     (new ClaimItemReturnedRequestBuilder()).forLoan(loan.getId().toString());
+
+    Response claimedLoan = loansFixture.claimItemReturned(claimedItemBuilder);
+    assertThat(claimedLoan, hasStatus(HTTP_NO_CONTENT));
+
+    Response fetchedLoan = loansClient.getById(loan.getId());
+
+    JsonObject loanToChange = fetchedLoan.getJson().copy();
+
+    final DateTime dueDate = DateTime.parse(loanToChange.getString("dueDate"));
+    final DateTime newDueDate = dueDate.plus(Period.days(14));
+
+    write(loanToChange, "action", "dueDateChange");
+    write(loanToChange, "dueDate", newDueDate);
+
+    Response updatedLoanResponse = loansFixture.attemptToReplaceLoan(loan.getId(), loanToChange);
+
+    assertThat(updatedLoanResponse.getStatusCode(), is(422));
+    assertThat("Should respond with appropriate due date change failed message",
+      updatedLoanResponse.getBody().contains("item is claimed returned"), is(true));
+
+    fetchedLoan = loansClient.getById(loan.getId());
+    loanToChange = fetchedLoan.getJson().copy();
+
+    write(loanToChange, "action", "dueDateChange");
+    write(loanToChange, "dueDate", dueDate);
+
+    loansFixture.replaceLoan(loan.getId(), loanToChange);
```

Comment:
What does this part of the test demonstrate? Is it meant to demonstrate that using the same due date for a due date change should be acceptable?

Reply:
The second test checks to see that the if the Due Date has changed to the same value.
In that case, it is technically an update (in regards to the HTTP request) to change the Due Date, but because the new date is the same as the previous date, the update should not fail due to a "due date change" while in  _Claimed Returned_ Item status.

Reply:
> The second test checks to see that the if the Due Date has changed to the same value.
> In that case, it is technically an update (in regards to the HTTP request) to change the Due 
Date, but because the new date is the same as the previous date, the update should not fail due to a "due date change" while in _Claimed Returned_ Item status.

I think that is an interesting edge case. If we want to demonstrate it in tests, I think it is could be more expressive for it be a separate test.

It does raise a follow-up question, which is should it be possible to attempt to change a due date to the same as the current due date?

I think this is exacerbated by the current API design, which makes it difficult to distinguish.

Reply:
The current design I used only checks to see if the due date has changed while in the specified Item status.
It has no way of knowing the nature of the update and would need to be more intelligent to handle the edge case as such.

As it stands, the edge case is a normal situation in the presented code, even if from the perspective of the user or frontend it is an edge case.

I think it should be fine making this a second, separate, test.